apps/challenges/urls.py

```diff
@@ -40,7 +40,8 @@
     ),
     # `A-Za-z` because it accepts either of `all, future, past or present` in either case
     url(
-        r"^challenge/(?P<challenge_time>[A-Za-z]+)/(?P<challenge_approved>[A-Za-z]+)/(?P<challenge_published>[A-Za-z]+)$",
+        r"^challenge/time/(?P<challenge_time>[A-Za-z]+)/approved/"
+        "(?P<challenge_approved>[A-Za-z]+)/visibility/(?P<challenge_published>[A-Za-z]+)$",
```

Comment:
Sorry for the back and forth. Looking at this API it isn't really readable. Can we instead convert it to a POST and GET API which has [this URL](https://github.com/Cloud-CV/EvalAI/commit/94f844952c4dba76c3cbe2b6929a978d7fc746d4#diff-7f035da148c99adb9058e04048f2cdb0e702f5eaa05c56f6a17adbbef702e4eeL43). 

And we will pass `is_approved` and `is_published` as request data for our cron jobs. Switching back to the old API URL will fix the existing breaking CLI and we just need to modify the cron job API calls and handle params from request data